packages/pwa-kit-dev/src/ssr/server/build-dev-server.js

```diff
@@ -126,6 +127,44 @@ export const DevServerMixin = {
         })
     },
 
+    /**
+     * @private
+     */
+    _setupExtensions(app, options) {
+        // TODO: support extensions options array syntax i.e. ['extension-a', {}]
+        const extensions = options.mobify?.app?.extensions || []
+        app.__extensions = extensions || []
+
+        extensions.forEach((extension) => {
+            const setupServerFilePathBase = path.join(
+                options.projectDir,
+                'node_modules',
+                extension,
+                'setup-server'
+            )
```

Comment:
In this PR, only extensions that are installed in `node_modules` is supported. I did not add support for relative file paths.

Reply:
If you plan is not to support this in this PR do you mind creating a ticket for that titled something like "Support for relative/local extensions in dev server extension loading" 🙏

Reply:
ticket created :)